### ISSUE TITLE
Resource location issue

### ISSUE THREAD

**Issue by sauterl** [vitrivr/cineast#80]
`SURFMirflickr25K256` feature cannot load the resource file as it is not found.

Branch: `multi-module`

**Comment by lucaro** [comment]
It expects a resources folder to be present at './resources'. The current build process does apparently not copy these resources properly. We will have to add that...

**Comment by silvanheller** [comment]
Instead of copying some resource folder, the ideal way is to put them in src/main/resources where they're automatically copied to the jar. This simplifies the build process.

**Comment by sauterl** [comment]
@silvanheller they are located in `cineast-core/src/main/resources/` - however, adjusting the [`CODEBOOK_FOLDER`](https://github.com/vitrivr/cineast/blob/multi-module/cineast-core/src/main/java/org/vitrivr/cineast/core/features/abstracts/AbstractCodebookFeatureModule.java#L37) to `codebooks/` does not fix the issue.

TL;DR: As @lucaro pointed out, the codebook files shall be copied to an accessible location during build

**Comment by lucaro** [comment]
We did in the past explicitly not copy these kinds of resources into the jar since they are very large which would bloat the jar and slow down the loading of everything and since they are (to a certain degree) user configurable. The code books for example can be generated via cineast, so if someone wants to generate one for their specific use case, they can do so. Same is true for the DNNs.

**Comment by silvanheller** [comment]
@sauterl is this still reproducible?
